docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -35,6 +35,57 @@ The system repeatedly shows a warning message about overriding an inherited asse
 
 This warning, appropriate for the first assessment, incorrectly reappears on subsequent reassessments, suggesting that the application is still inheriting its assessment, even after it has been overridden. link:https://issues.redhat.com/browse/MTA-1825[MTA-1825]
 
+.Custom rule for javax.{*} package import is not triggered when uploaded as an XML file
+
+In {ProductShortName} 7.0.0, some XML custom rule files are converted to the new YAML format during analysis. The root cause of this issue is Java search patterns that use `IMPORT` as `location`, should not end in `.{\*}` but should end in `{*}`. link:https://issues.redhat.com/browse/MTA-2000[MTA-2000]
+
+To resolve this issue, whenever a custom rule has a pattern ending in `.{\*}` and location `IMPORT`, the pattern `.{*}` can be changed to `{*}` and the location changed to `PACKAGE`.
+
+In the following example XML rule, note that the `when` condition is `javaclass references ="javax.{*}"`. 
+
+If you amend this to `javaclass references="javax{*}"`, the rule converts correctly.
+
```

Comment:
It looks redundant to me and it is missing the change from import to package

I recommend to omit this and end the previous line with "as shown in the following example"

Reply:
Clarifying:
lines 44-46 can be deleted